sys/cheri/cheric.h

```diff
@@ -92,6 +91,27 @@
 	(cheri_getbase(c) + cheri_getlen(c));			\
 })
 
+#ifdef NOTYET
+#define	cheri_is_subset(x, y)	__builtin_cheri_subset_test(x, y)
+#else
```

Comment:
I'd remove this as when we have the intrinsic we'll want to just revert the commit so the macro is back up where it was before.

Reply:
When we revert it we should change the macro parameters to be more descriptive.